<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>debugdialog</class>
 <widget class="QMainWindow" name="debugdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>371</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>阈值设置</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2" colspan="2">
      <widget class="QLineEdit" name="circularity_min">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>21</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>圆度范围</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>——</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="circularity_max">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>21</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>第一次处理</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>——</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QLineEdit" name="Threshold1_max">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>21</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLineEdit" name="area2">
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>21</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="Threshold1_min">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>21</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="5">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>第二次处理</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLineEdit" name="area1">
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>21</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="8">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>open</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>判断范围阈值</string>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QLineEdit" name="Threshold2_max">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>21</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLineEdit" name="Threshold2_min">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>21</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>判断是否异常阈值</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>——</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>roi区域</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QPushButton" name="pushButton_roi">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>重选</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="8">
      <widget class="QPushButton" name="pushButton_change">
       <property name="text">
        <string>change</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>421</width>
      <height>381</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>debugdialog</receiver>
   <slot>debugparam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>788</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>795</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_roi</sender>
   <signal>clicked()</signal>
   <receiver>debugdialog</receiver>
   <slot>getRoi()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_change</sender>
   <signal>clicked()</signal>
   <receiver>debugdialog</receiver>
   <slot>change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>751</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>772</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>debugparam()</slot>
  <slot>getRoi()</slot>
  <slot>change()</slot>
 </slots>
</ui>
